docs/multi-tenancy-in-minio.md

```diff
@@ -68,6 +68,10 @@ server {
            # proxy the request to Minio-3
 	       proxy_pass http://localhost:9003;
 	    }
+     }
+    location /ROML2P775VPAT7RLPOWU/ {
+      proxy_set_header    Host $http_host;
+      proxy_pass          http://127.0.0.1:9000/;
```

Comment:
This is the right config:
```
	   location ~* ^/ROML2P775VPAT7RLPOWU/(.*)$ {
	   	rewrite ^/ROML2P775VPAT7RLPOWU/(.*)$ /minio/$1 break;
	   	proxy_redirect     off;
	   	proxy_set_header Host $http_host;
	   	proxy_pass http://localhost:9001;
	   }
```

Adding this config here will make it harder to be grasped . Because most of the times users would not are not worried about accessing UI of multiple instances behind nginx.

What we can do is add a different section `Accessing web-UI of multiple minio instances running behind reverse proxy` and provide this configuration there:
```
server {
    listen       80;
	location / {
	    proxy_set_header Host $http_host;
	   location ~* ^/ROML2P775VPAT7RLPOWU/(.*)$ {
	   	rewrite ^/ROML2P775VPAT7RLPOWU/(.*)$ /minio/$1 break;
	   	proxy_redirect     off;
	   	proxy_set_header Host $http_host;
	   	proxy_pass http://localhost:9001;
	   }
	   location ~* ^/ENT3GYJCKCD1Q79XLP4C/(.*)$ {
	   	rewrite ^/ENT3GYJCKCD1Q79XLP4C/(.*)$ /minio/$1 break;
	   	proxy_redirect     off;
	   	proxy_set_header Host $http_host;
	   	proxy_pass http://localhost:9002;
	   }
	   location ~* ^/C988WQ23D98207ELOLPW/(.*)$ {
	   	rewrite ^/C988WQ23D98207ELOLPW/(.*)$ /minio/$1 break;
	   	proxy_redirect     off;
	   	proxy_set_header Host $http_host;
	   	proxy_pass http://localhost:9003;
	   }
     }
}
```
And mention in the doc saying that the web-UI can be accessed using the URLs:
http://nginxhost.com/ROML2P775VPAT7RLPOWU/
http://nginxhost.com/ENT3GYJCKCD1Q79XLP4C/
http://nginxhost.com/C988WQ23D98207ELOLPW/